include/fg/CPUCopy.hpp

```diff
@@ -0,0 +1,43 @@
+/*******************************************************
```

Comment:
Move these headers into include/ not include/fg. fg should contain only internal headers.